src/client/pages/AdminPage/index.tsx

```diff
@@ -51,10 +75,18 @@ const AdminPage: React.FunctionComponent = () => {
   return (
     <div className="container-fluid">
       <div className="row">
-        <div className="col-12">
-          <Action className="text-white shadow-sm" to="/admin/new">
+        <div className="col-12" style = {barStyle}>
```

Comment:
To answer your question = we style with styled-components. See `Action` on line 11 in this file.